src/frontend/src/components/ChangesList.tsx

```diff
@@ -26,21 +27,11 @@ const ChangesList: React.FC<ChangesListProps> = ({ changes }) => {
           <Link component={RouterLink} to={`${routes.CHANGE_REQUESTS}/${ic.changeRequestId}`}>
             #{ic.changeRequestId}
           </Link>
-          ]{' '}
-          <Tooltip
-            id="tooltip"
-            title={
-              <>
-                <Typography variant="body2">
-                  {fullNamePipe(ic.implementer)} - {datePipe(ic.dateImplemented)}
-                </Typography>
-              </>
-            }
-            placement="right"
-            arrow
-          >
-            <Typography component="span">{ic.detail}</Typography>
-          </Tooltip>
+          ]
+          <DynamicTooltip
+            title={`${fullNamePipe(ic.implementer)} - ${datePipe(ic.dateImplemented)}`}
+            children={<Typography component="span">{ic.detail}</Typography>}
```

Comment:
instead of declaring children as a prop, you declare children by making them child components of the parent. here, you would do:

`<DynamicTooltip ...>children go here</DynamicTooltip>`